<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>256</width>
      <height>551</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <row/>
    <row/>
    <column/>
   </widget>
   <widget class="QPushButton" name="pushButton_addrow">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить элемент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delrow">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать элемент</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_random">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Автозаполнение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_magik">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Сортировки</string>
    </property>
    <widget class="QRadioButton" name="radioButton_puz">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пузырьком</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_fast">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Быстрая</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_gnom">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Гномья</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rasch">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Расчёска</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_saru">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>199</y>
       <width>95</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Обезьянья</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимальное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимальное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mid">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_mid">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
